velox/expression/VectorUdfTypeSystem.h

```diff
@@ -611,6 +615,87 @@ struct VectorWriter<ArrayProxyT<V>> {
   vector_size_t offset_ = 0;
 };
 
+// A new temporary vector writer, to be used when the user wants a proxy
+// interface for the array output. Will eventually replace VectorWriter<Map>>.
```

Comment:
I think you forgot to remove the proxy part from this comment

// A new temporary vector writer for the new map writer interface.
// Will eventually replace VectorWriter<Map>>.